### ISSUE TITLE
Inconsistent memory format

### ISSUE THREAD

**Issue by tutenkum** [Luncher91/A2LParser#24]
While playing around with the export function I've found an issue for the AxisPts class. The output of the memory address is unlike other classes a decimal value instead of a hex value. The code snippet show the current AxisPts implementation and how to fix.
```
...
    writer.writelnBeginSpaced("AXIS_PTS", 
      name, 
      A2LWriter.toA2LString(longIdentifier), 
      Long.toString(address), // should be "0x" + Long.toHexString(address),
      inputQuantitiy,
      deposit,
      Double.toString(maxDiff),
      conversion,
      Long.toString(maxAxisPoints),
      Double.toString(lowerLimit),
      Double.toString(upperLimit));
...
```

**Comment by Luncher91** [comment]
Thank you for bringing that up. I will fix it 🙂

**Comment by Luncher91** [comment]
Change is included in Release 2.4.2
